web/new_playground.html

```diff
@@ -56,16 +56,28 @@
         <span>DartPad</span>
     </div>
     <div>
-        <button type="button" id="new-button" class="mdc-button">New Pad</button>
-        <button type="button" id="reset-button" class="mdc-button">Reset</button>
-        <button type="button" id="format-button" class="mdc-button">Format</button>
+        <button type="button" id="new-button" class="mdc-button">
+            <i class="material-icons mdc-button__icon">code</i>
+            New Pad
+        </button>
+        <button type="button" id="reset-button" class="mdc-button">
+            <i class="material-icons mdc-button__icon">refresh</i>
+            Reset
+        </button>
+        <button type="button" id="format-button" class="mdc-button">
+            <i class="material-icons mdc-button__icon">format_align_left</i>
+            Format
+        </button>
     </div>
     <div class="header-gist-name"></div>
     <div>
         <button type="button" id="share-button" class="mdc-button">Share</button>
     </div>
     <div>
-        <button type="button" id="samples-dropdown-button" class="mdc-button">Samples</button>
+        <button type="button" id="samples-dropdown-button" class="mdc-button">
+            <span class="mdc-button__label">Samples</span>
+            <i class="material-icons mdc-button__icon" aria-hidden="true">expand_more</i>
```

Comment:
This is the only aria-* attribute in the PR. Is it necessary for every user, or just an instance of the kind of accessibility metadata we should be putting everywhere?

Reply:
this is coming from the material web components docs. issue created here: https://github.com/dart-lang/dart-pad/issues/1297